### PR TITLE
Feedback implementation 

### PR DESCRIPTION
**Pull request by ravimeijerrig**
# Description
With this PR we implement the following feedback: 
- add sources to question 4.0 and 4.1
- Switch 5.1.0 and 5.1.1.
- Alter some use of language 
- Add context to conclusion 13.2.3 en 13.2.1

Resolves #

### CODE REVIEW

File: decision-tree.yaml [hunk 1]
```diff
@@ -923,15 +962,21 @@ conclusions:
       Wel moet deze AI-technologie uiteraard voldoen aan algemene veiligheidsstandaarden."
   - conclusionId: "13.2.1"
     conclusion: "Je bent een gebruiksverantwoordelijke van een AI-model voor algemene doeleinden. Er is sprake van een systeemrisico."
-    obligation: Als gebruiksverantwoordelijke van een AI-model voor algemene doeleinden gelden er geen verplichtingen vanuit het systeemrisico.
+    obligation: "Als gebruiksverantwoordelijke van een AI-model voor algemene doeleinden gelden er geen verplichtingen vanuit het systeemrisico.
+      Het is echter belangrijk om te realiseren dat als een GPAI-model verder wordt ontwikkeld, er mogelijk minimale transparantieverplichtingen van toepassing zijn.
```
Comment (GuusjeJuijn) [inline decision-tree.yaml:966]: Maybe change GPAI-model here into 'AI-model voor algemene doeleinden' for consistency?

File: decision-tree.yaml [hunk 2]
```diff
@@ -923,15 +962,21 @@ conclusions:
...
     sources:
       - source: Artikel 51 - Classificatie van AI-modellen voor algemene doeleinden als AI-modellen voor algemene doeleinden met een systeemrisico
         url: https://eur-lex.europa.eu/legal-content/NL/TXT/HTML/?uri=OJ:L_202401689#d1e5489-1-1
       - source: Begrippenlijst Algoritmekader
         url: https://minbzk.github.io/Algoritmekader/overhetalgoritmekader/definities/#begrippenlijst
   - conclusionId: "13.2.3"
     conclusion: "Je bent een gebruiksverantwoordelijke van een AI-model voor algemene doeleinden."
-    obligation: "Voor gebruiksverantwoordelijken van een AI-model voor algemene doeleinden, gelden er geen bijzondere vereisten vanuit de AI-verordening."
+    obligation: "Voor gebruiksverantwoordelijken van een AI-model voor algemene doeleinden, gelden er geen bijzondere vereisten vanuit de AI-verordening.
+      Het is echter belangrijk om te realiseren dat als een GPAI-model verder wordt ontwikkeld, er mogelijk minimale transparantieverplichtingen van toepassing zijn.
```
Comment (GuusjeJuijn) [inline decision-tree.yaml:977]: Same as above, GPAI model :)

File: decision-tree.yaml (hunk 1 above)
Comment (GuusjeJuijn) [inline decision-tree.yaml:966]: Maybe add here that if you change a GPAI model, it can become a GPAI system (and thus risk categories may apply)

**Review by GuusjeJuijn — APPROVED** [review]
LGTM, some small comments

**Review by laurensWe — APPROVED** [review]
Nothing to remark :)

**Pull request merged**